tox.ini

```
@@ -29,3 +29,10 @@ pass_env =
 commands =
     pre-commit run --all-files --show-diff-on-failure
     check-manifest
+
+[testenv:mypy]
+skip_install = true
+deps =
+    mypy==1.7.1
```

```suggestion
    mypy==1.7.1
    numpy
```
There is [currently](https://github.com/python-pillow/Pillow/actions/runs/7258653852/job/19774332621?pr=7622) a failure on ImageFilter.

A simple solution would be to exclude ImageFilter again, but this is also a fix.

---

This is also a good demo of tox: when I updated the list I had been running mypy directly, and have NumPy installed globally. If I'd run it via tox, I would have run into this failure :)